### DISCUSSION TITLE
key bindings send bytes

### DISCUSSION THREAD

**Discussion by max-tern** [raphamorim/rio#837] [Q&A] [answered]
Hi! Trying to migrate from alacritty to Rio and stuck with sending "bytes" to terminal. I want to close tmux session with cmd+w binding. In Alacritty it was { key = 'W', mods = "Command", chars = "\u0002x" } but not sure how to convert it to bytes?

**Comment by vinoff** [comment]
I am also having this issue. I opened an issue:

https://github.com/raphamorim/rio/issues/861

Did you figure it out?

**Comment by raphamorim** [comment, marked as answer]
hey there! both problems are different, I will convert this to an issue and try to add `chars` in the next version

@vinoff your problem is that Rio has a bug yet, that you need to disable and then add the key binding.

**Reply by vinoff** [reply]
What do I need to disable? The bug?

**Comment by raphamorim** [comment]
also feel free to join our discord https://discord.gg/zRvJjmKGwS

**Reply by hilaolu** [reply]
hi, the link seems not work for me.
